package.json

```diff
@@ -90,7 +89,7 @@
     "eth-ens-namehash": "2.0.8",
     "eth-provider": "0.13.6",
     "eth-sig-util": "3.0.1",
-    "ethereum-public-key-to-address": "0.0.5",
+    "ethereumjs-util": "^7.1.5",
```

Comment:
is this a new dependency? also for any dependencies we ship we want to remove the leading caret to protect against any new dependencies introducing vulnerabilities without us explicitly upgrading